packages/core/engine/src/index.ts

```diff
@@ -122,10 +122,12 @@ export class Engine {
   }
 
   async generateEmbarkJSON() {
-    if (fs.existsSync('embark.json')) {
-      throw new Error(__('embark.json already there. Will not overwrite'));
+    if (fs.existsSync('embark.json') || fs.existsSync('embark.config.js')) {
+      throw new Error(__('embark.json or embark.config.js already exist. Will not overwrite'));
+    }
+    if (!fs.existsSync('embark.config.js')) {
```

Comment:
This condition seems useless since you already check the same above

Reply:
Ah, good catch!